backend/seed_data/badges.yaml

```
@@ -18,7 +18,20 @@ badges:
     criteria_expression: "user_unit_completed == True and unit_title == 'Binary Trees'"
     event_type: UNIT_COMPLETION
 
+  - title: "Strings and Arrays"
+    description: "Awarded for completing the strings and arrays unit."
+    type: CONTENT
+    criteria_expression: "user_unit_completed == True and unit_title == 'Strings and Arrays'"
+    event_type: UNIT_COMPLETION
+
 # Award Badges
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)